db_test.go

```diff
@@ -1117,6 +1118,93 @@ func TestTypeAlias_inMappedStruct(t *testing.T) {
 	}
 }
 
+const versionedUsersDDL = `
+CREATE TABLE users (
+  id   BIGINT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  ver  INT NOT NULL
```

Comment:
Not necessarily: if you use an `int` for your version field, it would default to `0` in-memory and be inserted by squalor.